modules/bibauthorid/lib/bibauthorid_dbinterface.py

```diff
@@ -4105,8 +4121,23 @@ def bib_dictionarize(bibtable):
     MARC_100_700_CACHE['b100'] = bibd_10x
     MARC_100_700_CACHE['b700'] = bibd_70x
 
+    if (create_inverted_dicts or 'inverted_b100' in MARC_100_700_CACHE):
+        #MARC_100_700_CACHE['reversed_b100'] = dict((v,k) for k,v in MARC_100_700_CACHE['b100'].items())
+        MARC_100_700_CACHE['inverted_b100'] = dict(izip(MARC_100_700_CACHE['b100'].itervalues(), MARC_100_700_CACHE['b100'].iterkeys()))
+        #MARC_100_700_CACHE['reversed_b700'] = dict((v,k) for k,v in MARC_100_700_CACHE['b700'].items())
+        MARC_100_700_CACHE['inverted_b700'] = dict(izip(MARC_100_700_CACHE['b700'].itervalues(), MARC_100_700_CACHE['b700'].iterkeys()))
```

Comment:
Please remove comments. It would be also nice to comment what is this section doing.